src/mito/src/engine/procedure/create.rs

```diff
@@ -232,15 +232,22 @@ impl<S: StorageEngine> TableCreator<S> {
         let table_options = &self.data.request.table_options;
         let write_buffer_size = table_options.write_buffer_size.map(|size| size.0 as usize);
         let ttl = table_options.ttl;
+        let compaction_strategy = CompactionStrategy::from(&table_options.extra_options);
```

Comment:
question: Do we have a global option to set the compaction strategy for all tables?

Reply:
> question: Do we have a global option to set the compaction strategy for all tables?

The default compaction strategy is Leveled

Reply:
I think what @evenyag  meant is whether we have a `global_compaction` option in storage to configure all table's default strategies.